### DISCUSSION TITLE
StarDeck Module and Theme Designer

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#4251] [Show and tell]
Introducing StarDeck: your all-in-one solution for Module and Theme design within your Oqtane framework. With StarDeck, you're not just building a website – you're crafting an immersive digital experience from the ground up. Empowered by its seamless integration and comprehensive toolkit, StarDeck enables you to shape every aspect of your site's appearance and functionality, starting from the theme level. Take control of your Oqtane solution and unleash your creativity.

In StarDeck, creating a page layout is very simple on the fly.
![stardeckPageLayout](https://github.com/oqtane/oqtane.framework/assets/9447612/512f2aa0-c368-4979-abd7-a39552cac7d0)

Imagine effortlessly transforming your HTML text modules into dynamic StarDeck modules with just a click. No more tedious editing, copying, or pasting – StarDeck streamlines the process, saving you valuable time and effort. And the best part? You can seamlessly convert them back whenever you need. It's like magic for your content creation workflow!
![stardeckConvertTo](https://github.com/oqtane/oqtane.framework/assets/9447612/79e8a9eb-d10e-4285-b488-9ef0425b29f9)

Discover the versatility of StarDeck modules as they seamlessly template your containers. By adding a class to a tag, you can effortlessly StarJump your content exactly where you want it within the container. It's a simple yet effective way to organize your layout with precision. Explore the convenience of this feature and elevate your content creation process with ease.
![stardeckJumpToContainer](https://github.com/oqtane/oqtane.framework/assets/9447612/926f57de-1394-4b87-8965-348b1be7d31c)

**Lets, Build**
Utilizing the Arsha template obtained from https://bootstrapmade.com/arsha-free-bootstrap-html-template-corporate/, we can swiftly recreate it in StarDeck. Here is the rendition of the Hero section.
![stardeckBuildHero](https://github.com/oqtane/oqtane.framework/assets/9447612/4b027804-a873-4250-89ed-b6d0c9f94bfc)

Hero Section fully styled.
![stardeckBuildHeroStyle](https://github.com/oqtane/oqtane.framework/assets/9447612/f8d7d6f8-5ae8-49bc-a1c0-babcb3c59e1a)

**Comment by jaruiz-ual** [comment]
Please,
I'd like to test StarDesk, I'm very interesting.
How can I get it?
Thank you.
